src/countries/my.js

```diff
@@ -10,27 +20,31 @@ var config = {
   }
 }
 
-var fixMyShortPhone = tools.fixPhoneBuilder(9, 11, config);
-var fixMyLongPhone = tools.fixPhoneBuilder(9, 12, config);
-var fixMyMobilePhone = tools.fixPhoneBuilder(10, 13, config);
+var fixMyShortPhone = tools.fixPhoneBuilder(9, 12, shortPhoneConfig);
+var fixMyLongPhone = tools.fixPhoneBuilder(10, 13, longPhoneConfig);
 
 var fixMyPhone = function (phone) {
+  phone = phone.replace(/^(0{1,2})/, '')
   phone = tools.getSanitizedPhone(phone)
-
-  var localCode = tools.getLocalCode(config, phone)
-
-  if (localCode.length == 2 && phone.length === 13) {
-    return fixMyMobilePhone(phone)
-  }
-  if (localCode.length < 2 && phone.length === 12) {
+  var localCode = tools.getLocalCode(shortPhoneConfig, phone)
+  if (localCode.indexOf('11') === 0) {
+    localCode = tools.getLocalCode(longPhoneConfig, phone)
     return fixMyLongPhone(phone)
-  }
-  if (localCode.length < 2 && phone.length === 11) {
+  } else {
     return fixMyShortPhone(phone)
   }
+} 
+
+function decompose (phone) {
+  var decompose = tools.decomposeBuilder(fixMyPhone, longPhoneConfig)(phone)
```

Comment:
`decompose` is bad var name (verb)
`tools.decomposeBuilder(fixMyPhone, longPhoneConfig)` can be done once, instead of doing it each time function is called